style.css

```diff
@@ -0,0 +1,186 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto:wght@100;400&display=swap');
+
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-wrapper {
+  max-width: 400px;
+  height: 92px;
+  background: #3c3a39;
+  display: flex;
+}
+
+.navtop {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 10px 10px;
+  color: #fff;
+  position: absolute;
+  width: 400px;
+  height: 44px;
+}
+
+.time {
+  height: 16px;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: center;
+  letter-spacing: -0.28px;
+}
+
+.navbottom {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  color: #fff;
+  position: absolute;
+  width: 363px;
+  top: 44px;
+}
+
+.logo {
+  position: static;
+  height: 24px;
+  left: 12px;
+  top: 10px;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 12px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+}
```

Comment:
- [ ] Let's try to add padding to the left of the logo so it will match the Figma design

<img width="295" alt="image" src="https://user-images.githubusercontent.com/77190203/148376727-8419324a-ea0d-49b2-b502-b5a6bc78b764.png">